packages/legend-studio/src/stores/ModifierHelper.ts

```diff
@@ -0,0 +1,561 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type {
+  Class,
+  FlatDataConnection,
+  FlatDataInstanceSetImplementation,
+  InstanceSetImplementation,
+  JsonModelConnection,
+  KeyedExecutionParameter,
+  Multiplicity,
+  PropertyMapping,
+  PureExecution,
+  PureMultiExecution,
+  PureSingleExecution,
+  RawLambda,
+  RawVariableExpression,
+  RootFlatDataRecordType,
+  Runtime,
+  SetImplementation,
+  Type,
+  XmlModelConnection,
+} from '@finos/legend-graph';
+import {
+  type Binding,
+  type Connection,
+  type ExternalFormatSchema,
+  type ModelUnit,
+  type PackageableConnection,
+  type PackageableElement,
+  type PackageableElementReference,
+  type SchemaSet,
+  type Store,
+  type UrlStream,
+  type BindingTransformer,
+  type ConfigurationProperty,
+  type GenerationSpecification,
+  type PackageableRuntime,
+  type EngineRuntime,
+  type RuntimePointer,
+  FileGenerationSpecification,
+  GenerationTreeNode,
+  PackageableElementExplicitReference,
+  ModelGenerationSpecification,
+  DEFAULT_SERVICE_PATTERN,
+  type IdentifiedConnection,
+  type Mapping,
+  type Service,
+  type ServiceExecution,
+  type ServiceTest,
+  StoreConnections,
+} from '@finos/legend-graph';
+import {
+  addUniqueEntry,
+  assertTrue,
+  changeEntry,
+  deleteEntry,
+  UnsupportedOperationError,
+  uuid,
+} from '@finos/legend-shared';
+import { action } from 'mobx';
+
+// Connection
```

Comment:
I think we could safely split them here into `DSLService_ModifierHelper`, `DSLMApping_ModifierHelper`, etc.